### ISSUE TITLE
`prefer-math-min-max`: ignore bigint variable

### ISSUE THREAD

**Issue by yoursunny** [sindresorhus/eslint-plugin-unicorn#2509]
**unicorn/prefer-math-min-max** is incorrectly reporting variables of bigint type.
Its autofix generates invalid code.

## package.json

```json
{
  "private": true,
  "type": "module",
  "packageManager": "pnpm@9.15.0+sha512.76e2379760a4328ec4415815bcd6628dee727af3779aaa4c914e3944156c4299921a89f976381ee107d41f12cfa4b66681ca9c718f0668fa0831ed4c6d8ba56c",
  "dependencies": {
    "typescript": "~5.7.2",
    "xo": "^0.60.0"
  }
}
```

## tsconfig.json

```jsonc
{
  "compilerOptions": {
    "forceConsistentCasingInFileNames": true,
    "lib": [
      "DOM",
      "ES2023"
    ],
    "module": "ES2022",
    "moduleResolution": "Bundler",
    "noEmitOnError": true,
    "noImplicitOverride": true,
    "noImplicitReturns": true,
    "noUncheckedIndexedAccess": true,
    "noUnusedLocals": true,
    "strict": true,
    "target": "ES2023"
  },
  "include": [
    "main.ts"
  ],
  "exclude": [
    "**/node_modules"
  ]
}
```

## main.ts

```ts
export function f(a: bigint, b = 0n) {
	return a > b ? a : b;
}
```

## Command Line

```bash
./node_modules/.bin/xo
```

## Error Message

```text
  main.ts:2:9
  ✖  2:9  Prefer Math.max() to simplify ternary expressions.  unicorn/prefer-math-min-max

  1 error
```

## Autofix Result and TypeScript Error

```ts
export function f(a: bigint, b = 0n) {
	return Math.max(a, b);
}
```

```text
main.ts:2:18 - error TS2345: Argument of type 'bigint' is not assignable to parameter of type 'number'.

2  return Math.max(a, b);
                   ~


Found 1 error in main.ts:2
```
